pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -543,3 +562,34 @@ func (i *Gatherer) ClusterVersion() *configv1.ClusterVersion {
 	defer i.lock.Unlock()
 	return i.lastVersion
 }
+
+// ConfigMapAnonymizer implements serialization of configmap
+// and potentially anonymizes if it is a certificate
+type ConfigMapAnonymizer struct{ v []byte }
+
+// Marshal implements serialization of Node with anonymization
+func (a ConfigMapAnonymizer) Marshal(_ context.Context) ([]byte, error) {
+	return []byte(anonymizeConfigMap(a.v)), nil
+}
+
+func anonymizeConfigMap(dv []byte) string {
+	anonymizedPemBlock := `-----BEGIN CERTIFICATE-----
+YW5vbnltaXplZA==
```

Comment:
Why specifically this string?

Reply:
I wanted it to still be a valid PEM, the "YW5vbnltaXplZA==" is base64 of  "anonymized" in case anyone would be trying to decode it.

Reply:
My first reaction was to try using `openssl` against it and it failed terribly. So just mentioning `ANONYMIZED`, or `XXXXXXXXXXXXXX` would be feel intuitive to me